### ISSUE TITLE
Make stroke IDs unique in eyedraw

### ISSUE THREAD

**Issue by andyqhan** [pomonahci/EyeWrite#13]
May solve some weird bugs

**Comment by andyqhan** [comment]
All the relevant code, I think, is in `draw-syncing.js`. However, the way it currently works is to use `ret = primSket.currentPath.serialize()`. The object (`ret`) returned by that function contains an attribute, `idStroke`, which I think is the ID of the stroke. The problem is that I can't find the definition of `primSket` or of that `serialize()` function. I've grepped for it in the branch and it seems like it's defined in `draw/main.af3e61f1.chunk.js`, which is, as the name suggests, illegible minified JavaScript. I can't find the source code for this file; I don't think it's in this repo. (It may also be the case that `ret.idStroke` is the same as `primSket.currentPath.idStroke`, since in line 152 we set `currentPath.idStroke` to a new value and the code seems to have the intention that this ID is passed through to `clickContent`.)

The immediate solution I'll implement for now is hacky: simply redefining `ret.idStroke` to the output of some UUID generator. I think that finding the source code for `draw/main.af3e61f1.chunk.js`, understanding it, changing it, then re-minifying it will be more work than it's worth, especially if our bugs aren't due to ID collision.
